### PR TITLE
Pop up windows feature

### PR DESCRIPTION
**Pull request by gergitokusse**
### The following file was added and modified

- Style.css file was updated for adjusting pop up window
- Same new html tag was added.
- jscript.js file was modified

### CODE REVIEW

File: jscript.js [hunk 1]
```diff
@@ -37,4 +36,226 @@ function closemenu() {
...
+      <p> Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+      Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown 
+      printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the 
+      printing and typesetting industry. </p><br>
+
+      <p>Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer 
+      took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing 
+      and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the 
+      releawn printer took a galley of type and scrambled it 1960s. </p>
+    </div>
```
Comment (AlphaNtihinduka) [inline jscript.js]: - [ ] You have done very nice work to have a good design and a functioning window popup. However, we need to implement the use of [array of objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#using_map_to_reformat_objects_in_an_array). Therefore the objects should have the key and value which is not HTML tags. Kindly consider following this example to refactor your array of objects and use them properly.

`
const projects = [{
    name: "tomic",
    description: "This is tonic description",
    image: "link_to_image.jpg",
    technologies: ["HTML", "JavaScript", "css"],
    link: "live link",
}]
`

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hello Team🙋‍♂️
Good job working on your window popup. You have done the design to the best expectation and incredibly well. 👏  
. 
You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: jscript.js (hunk 1 above)
Comment (gergitokusse) [reply jscript.js]: ### Okay!!! I will try to change it.

**Review by amejid — COMMENTED** [review]
Hi team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where ESLint check is not running. Kindly use the `linters.yml` file from the [HTML-CSS-JS](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) folder of the linters config repository.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by gergitokusse** [comment]
> Hi team,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, where ESLint check is not running. Kindly use the `linters.yml` file from the [HTML-CSS-JS](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) folder of the linters config repository.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

**Review by omarsalem7 — APPROVED** [review]
Hi @gergitokusse 😊, 
# STATUS: APPROVED! 🥇 💯 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ No linter errors. 
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**